wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -166,9 +166,8 @@ data class JavaTarget(
         val javaTypeName = javaGenerator.generatedTypeName(type)
         val javaFile = JavaFile.builder(javaTypeName.packageName(), typeSpec)
             .addFileComment("\$L", WireCompiler.CODE_GENERATED_BY_WIRE)
-            .apply {
```

Comment:
What was this apply doing here....

Reply:
I assume it had some conditional logic in it at one point. Dunno.